TreblleCore/TrebllePayloadFactory.cs

```diff
@@ -165,24 +149,29 @@ private async Task TryAddResponse(HttpContext httpContext, Stream? response, lon
             payload.Data.Response.Size = responseContent.Length;
         }
 
+        TryAddRequestHeaders(httpContext, payload);
+
+        payload.Data.Response.Code = httpContext.Response.StatusCode;
+        payload.Data.Response.LoadTime = elapsedMilliseconds / (double)1000;
+    }
+
+    private void TryAddRequestHeaders(HttpContext httpContext, TrebllePayload payload)
+    {
         try
         {
-            payload.Data.Response.Headers =
-                httpContext.Response.Headers.ToDictionary(x => x.Key, x => string.Join(";", x.Value));
+            payload.Data.Request.Headers =
+                httpContext.Request.Headers.ToDictionary(x => x.Key, x => string.Join(';', x.Value));
```

Comment:
Hey @VasilevichPavel,

I have reviewed all the files. Great Work!

Can you explain why are we changing to Request instead of Response here?